estate/models/estate_property.py

```diff
@@ -1,26 +1,43 @@
 from odoo import fields, models
+from dateutil.relativedelta import relativedelta
 
 class RealEstate(models.Model):
     # Model name and description
     _name = "estate.property"
     _description =  """
                         Estate model
                     """
-
     # Fields
     name = fields.Char(required=True)
-    description = fields.Text()
+    description = fields.Text() 
     postcode = fields.Char()
-    date_availability = fields.Date()
-    excepted_price = fields.Float(required=True)
-    selling_price = fields.Float()
-    bedrooms = fields.Float()
+
+    def _default_date(self):
+        return fields.Date.today() + relativedelta(months=3)
+
+    date_availability = fields.Date(copy = False, default = _default_date)
+    excepted_price = fields.Float(required = True)
+    selling_price = fields.Float(readonly = True, copy = False)
+    bedrooms = fields.Float(default = 2)
     living_area = fields.Integer()
     facades = fields.Integer()
     garage = fields.Boolean()
     garden = fields.Boolean()
     garden_area = fields.Integer()
     garden_orientation = fields.Selection([
-        ("north","North"), ("south", "South"), ("east", "East"), ("west", "West"),
+        ("north", "North"), ("south", "South"), ("east", "East"), ("west", "West"),
     ])
+    activate = fields.Boolean(default = True)
```

Comment:
C'est un champ bien particulier dans Odoo, il faut le nommer `active`
```suggestion
    active = fields.Boolean(default = True)
```